### ISSUE TITLE
[feature]: Extend (E-Mail) notification system

### ISSUE THREAD

**Issue by etaxi341** [makeplane/plane#3498]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Summary

E-Mail notification was added recently in dev and preview.

I would like to request another type of notification.
When a user is assigned to a ticket I think it should count the same as when he is subscribed.

Otherwise every user that works on a ticket has to subscribe to it for mail notifications when new comments are added.
Also the user should be notified when he gets assigned to a ticket.

### Why should this be worked on?

Better Workflow

**Comment by pablohashescobar** [comment]
@etaxi341, The behavior you described should already be in place: when a user creates or is assigned an issue, they are automatically subscribed to that issue. If you're not experiencing this, could you please confirm or provide more details on what you're observing?

**Comment by pablohashescobar** [comment]
I'm not sure I've fully understood your situation, but to clarify, are you not receiving an email notification when someone assigns you to an issue?

**Comment by etaxi341** [comment]
@pablohashescobar correct. When someone assigns me an issue I don't get notifications in the notification tab and also no email (I am currently on the most recent preview branch with working email notifications for subscribed issues)

**Comment by pablohashescobar** [comment]
That's strange just to confirm you are on the self hosted version right ? Also do you see any error logs for the worker container.

**Comment by etaxi341** [comment]
Yes I am on selfhosted.
There are no Error Logs in the worker container, but I found why it happens:

Case 1:
1. User creates a Ticket and does not assign anyone
2. User assigns me after the Ticket was created
3. WORKS! (I get a notification)

Case 2:
1. User creates Ticket and assigns me in the create screen
2. DOES NOT WORK (I dont get a notification)

It would be nice if case 2 would work too because some users already know who the issue should be assigned to.

**Comment by pablohashescobar** [comment]
Got it @etaxi341, I think we missed this case we will add it. Thanks for pointing out.

**Comment by purvalpatel** [comment]
My configuration is proper and email are working but it tooks 5 minutes to send mails.

`{"levelname": "INFO", "asctime": "2026-01-13 04:50:01,560", "module": "email_notification_task", "name": "plane.worker", "message": "Email Sent Successfully"}`
